src/structures/game_session.ts

```diff
@@ -619,6 +621,56 @@ export default class GameSession extends Session {
         await super.updateOwner();
     }
 
+    async handleClipComponentInteraction(
+        round: ClipGameRound,
+        interaction: Eris.ComponentInteraction<Eris.TextableChannel>,
+        messageContext: MessageContext,
+    ): Promise<boolean> {
+        if (!round.songStartedAt || round.getReplayCount() === 0) {
+            // Prevent spamming clip actions
+            await tryCreateInteractionErrorAcknowledgement(
+                interaction,
+                null,
+                i18n.translate(
+                    this.guildID,
+                    "misc.failure.interaction.clipActionTooEarly",
+                ),
+            );
+            return true;
+        }
+
+        const clipRound = round as ClipGameRound;
+        clipRound.newClipRequested(messageContext.author.id);
+        await tryCreateInteractionSuccessAcknowledgement(
+            interaction,
+            i18n.translate(
+                this.guildID,
+                clipRound.isNewClipMajority()
+                    ? "misc.newClip"
+                    : "misc.clip.newClip.request.title",
+            ),
+            i18n.translate(
+                this.guildID,
+                clipRound.isNewClipMajority()
+                    ? "misc.clip.newClip.success"
+                    : "misc.clip.newClip.request.description",
+                {
+                    voteCounter: clipRound.newClipVoteCounter(),
+                },
+            ),
+            !clipRound.isNewClipMajority(),
```

Comment:
Why are we making this conditionally ephermeral? Are hints/skips ephermeral?

Reply:
They are not ephemeral, we can make these non-ephemeral as well if you'd prefer. Was hoping to cut down on chat spam in a short time interval